src/ts/components/button.ts

```diff
@@ -71,7 +71,8 @@ export class Button<Config extends ButtonConfig> extends Component<Config> {
     }).html(i18n.performLocalization(this.config.text)));
 
     // Listen for the click event on the button element and trigger the corresponding event on the button component
-    buttonElement.on('click', () => {
+    buttonElement.on('click', (e) => {
+      e.stopPropagation();
```

Comment:
This is needed to prevent the event from bubbling up to the parent element. 

Example: In the new design, the whole `SettingsPanelItem` is clickable. The subtitle customization item also contains a button to a different page. Without this change, both would trigger when clicking the button.